samples/meetings-app-icon-badging/csharp/AppIconBadgingInMeetings/AppManifest/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```

Comment:
Why we've downgraded, from 1.16 to 1.14

Reply:
These sample use rsc permissions, which break in latest manifest version